sormas-api/src/main/java/de/symeda/sormas/api/symptoms/SymptomsHelper.java

```diff
@@ -195,7 +196,14 @@ public static void updateIsSymptomatic(SymptomsDto dto) {
 			throw new RuntimeException(e);
 		}
 
-		dto.setSymptomatic(false);
+		if (dto.getSymptomatic() == YesNoUnknown.UNKNOWN) {
```

Comment:
This code block doesn't really make sense; all you're doing is setting the same value that was already set before. Should be removed.